gwsumm/tabs/core.py

```diff
@@ -49,12 +49,23 @@
 from ..mode import (Mode, get_mode, get_base)
 from ..utils import (re_quote, re_cchar)
 from .registry import (get_tab, register_tab)
-from .._version import get_versions
 
 __author__ = 'Duncan Macleod <duncan.macleod@ligo.org>'
 __all__ = ['BaseTab', 'Tab', 'TabList']
 
 
+def get_version():
+    try:
+        from ._version import version as __version__
+    except ModuleNotFoundError:
+        try:
+            import setuptools_scm
+            __version__ = setuptools_scm.get_version(fallback_version='?.?.?')
+        except (ModuleNotFoundError, TypeError, LookupError):
+            __version__ = '?.?.?'
+    return __version__
```

Comment:
I would replace all of this with an import of the top-level package version

```python
from .. import __version__
```

and then just return it:

```suggestion
    return __version__
```

Reply:
Fixed